packages/obojobo-document-engine/ObojoboDraft/Chunks/IFrame/editor-component.js

```diff
@@ -1,74 +1,275 @@
 import React from 'react'
+import Common from 'Common'
```

Comment:
Similar to https://github.com/ucfopen/Obojobo/pull/671  this file has `id`s that need more specific selectors